action.yml

```diff
@@ -118,16 +129,18 @@ runs:
     - name: Download Maven S3 repository extension
       shell: bash
       run: |
+        TMP_WAGON_PATH="/tmp/s3-wagon-${{ inputs.maven-wagon-version }}-jar-with-dependencies.jar"
+        aws s3 cp ${{ env.MAVEN_WAGON_PATH }} $TMP_WAGON_PATH --profile ${{ steps.configure-maven-aws-profile.outputs.aws-profile }}
```

Comment:
In this way I workarounded that I do not need to have aws credentials in "sudo" context so we can use that profile outside of action. So I will only copy it with sudo on place which needs sudo access.